runtime/lua/nvim/_fix_list.lua

```diff
@@ -0,0 +1,66 @@
+-- TODO(tjdevries): Make the metamethods work for __len
+--                      I can't seem to get that to work the way I want it to.
```

Comment:
IIRC the `__len` metamethod is only in lua 5.2, but we're using 5.1, so another approach is needed.